recipes/icu/bld.bat

```diff
@@ -0,0 +1,15 @@
+if "%ARCH%"=="32" (
```

Comment:
Looks like only Python 3.4 builds with this recipe.  Whoops.  I must not have finished it.  I'll work on it.